index.html

```diff
@@ -345,13 +345,13 @@ <h3 class="about-title">About Me</h3>
                 </div>
 
                 <div class="contact-form">
-                    <form class="form-block" action="submit">
+                    <form class="form-block" action="https://formspree.io/f/xeqnpgqv" method="post">
```

Comment:
Why not use `<section>` tag for the contact form section. This make your code SEO friendly and semantically correct.